added functions from gravitational force to centripetal force

SagarJC opened the pull request.

SagarJC committed:
Update naxxatrapy.py

SagarJC committed:
added functions from gravitational force to centrepetal force

jee1mr commented:
@SagarJC In all the places, instead of storing the result first in another variable, please return the expression directly. Other than that, refer to the feedback given to sanath here #2 and make the same changes.

SagarJC committed:
Update naxxatrapy.py

SagarJC committed:
Update naxxatrapy.py

Added some more functions related to rotational mechanics like angular displacement, angular velocity, angular acceleration, angular momentum, moment of inertia, centripetal acceleration, angular kinetic energy and torque. Imported all functions from numpy and specified Gravitational constant.
